### PR TITLE
The net neutrality rollback and how skycoin can defeat it.ja.md

### PR DESCRIPTION
**Pull request by yamadaman**
宜しくお願い致します。

Sky address:
2JneX1kRBfdWzJk9oZjt5am1ougm944a2wd

### CODE REVIEW

**Comment by angryxueqiu** [comment]
One thing I would like to discuss with you guys: 

"中立性" is the usual translation of "neutrality".
But somehow this phrase is kind of confusing. 

So basically the context of "neutrality " is any data will be treated the same.  

I feel  "同権"  と "均権"  will bring it a little bit closer to the context.

What do you think? 

Thanks for the translation anyways 👍  :)

**Comment by ShawnTaylorJP** [comment]
Hi, angryxueqiu
Are you talking about Chinese word?
I don't think using "中立性" is confusing for the context in Japanese translation.
"同権" seems to be a word for only the word "男女同権" (Gender equality) in Japanese.
I've never heard of "均権" in any context.

**Comment by yamadaman** [comment]
I think that the word "中立性” is appropriate in this context.
We can see the content that "Ajit Pai is trying to break the principle of treating every site fairly,"at the news site,  Regarding this context, the Japanese word "中立性" can express meaning that "treating every website fairly."

**Comment by angryxueqiu** [comment]
Hi yamadaman and ShawnTaylorJP,

Cool. Thanks for the advice, i know "同権" is more like equality. Also ""均権" is just sooooooooo old LOL.

**Comment by yamadaman** [comment]
OK. Thanks for your advice!

File: content/statement/The Net Neutrality Rollback and How Skycoin Can Defeat It.ja.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+あなたは、抑圧、ブロッキング、または検閲されることなく自由にコンテンツを共有することができます。
+それは高速で、プライベートで、暗号化されており、最大限に安全です。
+加えて、 Skycoinネットワークの決定は、利益を求める企業ではなく地域社会のコンセンサスによって決まります。
+
+P2P技術では、あなたの身元やコンテンツにアクセスするためのデータを要求しません。 
+Skycoinのネットワーク、すなわちSkywireは、安全かつ個人的に閲覧することができるブロックチェーンベースの分散型ネットワークを提供します。
+Skywireを使用すると、あなたのデータは自分で管理でき、あなたの体験をコントロールできます。
+Skywireの "マイナー"は、この新しいインターネットのバックボーンです。 
+このマイナーはユーザーに、ネットワークノードの展開、ネットワークトラフィックの転送、またはリソースへの貢献によってSkycoinを獲得させます。
+Skycoinシステムは、ユーザーがネットワークの成長と実行可能性に責任を負う、フリーでコミュニティベースのモデルを奨励します。
```
Comment (tateshita19) [inline content/statement/The Net Neutrality Rollback and How Skycoin Can Defeat It.ja.md]: フリー→自由で
15,62,68行などたびたび登場する言葉なので統一しておいたほうがいいかと思います。

File: content/statement/The Net Neutrality Rollback and How Skycoin Can Defeat It.ja.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+今夏のPai委員長に対する批判への無反応は、ネット中立性の後退を食い止めるために、従来の手法に頼れないことを示しています。
+我々は別の解決策が必要です。 
+そこでSkycoinの登場です。
+
+Skycoinの技術は、新しいインターネットを作り出します。
+これは、ISPを回避して制御をユーザに戻すP2Pネットワークを提供します。 
+あなたは、抑圧、ブロッキング、または検閲されることなく自由にコンテンツを共有することができます。
+それは高速で、プライベートで、暗号化されており、最大限に安全です。
+加えて、 Skycoinネットワークの決定は、利益を求める企業ではなく地域社会のコンセンサスによって決まります。
```
Comment (tateshita19) [inline content/statement/The Net Neutrality Rollback and How Skycoin Can Defeat It.ja.md]: 地域社会のコンセンサス
→コミュニティの合意形成
はどうでしょうか。

File: content/statement/The Net Neutrality Rollback and How Skycoin Can Defeat It.ja.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+また、公共の利益として、健康から政策に至るまでのトピックの透明性を向上させることができます。
+芸術はより広範な層にアクセス可能となり、多様な考え方を聞くことが出来ます。
+ユーザーは、限られた数の企業製品を使用するよりも、独自のコンテンツを選択して作成することができます。
+
+フリーでオープンなインターネットはこれまで以上に攻撃を受けています。
+今年、FCCがネットの中立規制を支持しないのであれば、私たち次第で、コントロールを取り戻し、インターネットがすべての人に平等に利用できる公的資源であることを確認をすることができます。
+
+Skywireネットワークは、インターネットを探検するためにISPに頼る必要がないことを証明しています。 
+Skywireのプロトコルとオーダーメードのハードウェアは、公平でユーザー中心のグローバルな分散型ネットワークを提供します。
+このネットワークにより、Skywireコミュニティは、FCCの計画されたネット中立性の後退を打ち負かし、インターネットをあなたと他のユーザーに戻すことができます。
```
Comment (tateshita19) [inline content/statement/The Net Neutrality Rollback and How Skycoin Can Defeat It.ja.md]: FCCの計画されたネット中立性の後退を打ち負かし、
→FCCにより計画されたネット中立性の後退を打ち負かし、
はどうでしょうか。（なんとなくしっくりこないので）

**Comment by yamadaman** [comment]
Aprprove頂きまして、誠に有難うございます！

**Comment by gz-c** [comment]
6 sky

**Comment by yamadaman** [comment]
Thank you very much!

**Pull request merged**